docs/user_guide.rst

```diff
@@ -43,6 +43,10 @@ Specifying a Base URL
 
 To specify a base URL, refer to the documentation for the
 `pytest-base-url <https://github.com/pytest-dev/pytest-base-url>`_ plugin.
+Don't forget to configure the :ref:`sensitive environments <sensitive-environments>`
+or your tests will be skipped.
```

Comment:
Only destructive tests that use the base URL fixture will be skipped when the base URL matches a sensitive environment. I think it's important to make this clear. Perhaps reword this as:

```rst
Note: By default, any tests using a base URL will be skipped. This is because all tests
are considered destructive, and all environments are considered sensitive. See
`Sensitive Environments`_ for further details.
```